README.md

```diff
@@ -33,6 +33,7 @@ Usage of ./go-get-proxied:
   -c string
     	Optional. Path to configuration file.
   -j	Optional. If a proxy is found, write it as JSON instead of a URL.
+  -l	Optional. If set, a list of proxy will be returned.
```

Comment:
NIT: a list of "proxies"

Reply:
I'll fix it in the code instead of just readme.